tests/fixtures.py

```diff
@@ -36,6 +36,10 @@
             'name': {
                 'type': ['string']
             },
+            'paw_size': {
+                'type': ['integer'],
+                'default': 314159
```

Comment:
That's a big paw

Reply:
These cats have paws for pi and pi alone

Reply:
@AlexanderMann cat pi https://i.pinimg.com/originals/60/46/5d/60465da88007d8e5b28f265c7805abce.jpg